.circleci/config.yml

```diff
@@ -19,6 +19,21 @@ runtest: &runtest
 version: 2.1
 
 jobs:
+  type-check:
+    docker:
+      - image: awav/tensorflow:2.1
+
+    steps:
+      - checkout
+      - run:
+          name: Install mypy
+          command: |
+            pip install mypy
```

Comment:
Should this not be part of `tests_requirements.txt` instead?

Reply:
debatable - I don't consider type checking to be part of the tests, but then again neither would I consider black to be part of the tests and that's in there

won't it will slow down the type checking job to install the other stuff?

Reply:
I guess it should be "this should _also_ be part of `tests_requirements.txt`". We should make it easy for contributors to run tests locally!

Reply:
curiously, black's in test_requirements.txt _and_ installed separately like i've done for mypy

Reply:
but the whole thing's moot until I can get the type checking to appear on the checks below